test/sort.jl

```diff
@@ -92,11 +92,14 @@ for df_rand in [df_rand1, df_rand2]
     # testing if sort! is consistent with issorted and sort
     ref_df = df_rand[:]
```

Comment:
This should be enough, right?
```suggestion
    ref_df = df_rand
```

Also I find `copy` more explicit.